README.md

```diff
@@ -1,70 +1,64 @@
-# Getting Started with Create React App
+![](https://img.shields.io/badge/Microverse-blueviolet)
```

Comment:
- [ ] I don't think these boilerplate files would be relevant to this project. To keep the codebase clean, I would recommend you remove them and resolve resulting import issues.

  ![giuseppe-src](https://user-images.githubusercontent.com/54848363/176318405-43ab3104-b2fe-4b75-8ffd-5e1235fe09fa.jpg)